src/core/request.js

```diff
@@ -59,31 +59,31 @@ Request.prototype.request = function(servers, callback_) {
   this.on('error', function() {});
   this.emit('request', this.remote);
 
-  function doRequest() {
-    if (Array.isArray(servers)) {
-      servers.forEach(function(server) {
-        self.setServer(server);
-        self.remote.request(self);
-      }, self);
-    } else {
-      self.remote.request(self);
+  const timeout = setTimeout(() => {
+    if (typeof callback === 'function') {
+      callback(new RippleError('tejTimeout'));
     }
-  }
 
-  function onReconnect() {
```

Comment:
Actually, I'm not really convinced that we have to remove it even for transaction requests. We already interpret "tef" errors as "maybe succeeded", so it might be easiest to just leave it as it was. But I still think it would be good to keep the default timeout.